src/Akka.CQRS.Pricing.Service/Program.cs

```diff
@@ -48,58 +53,80 @@ public static async Task<int> Main(string[] args)
             await Task.Delay(TimeSpan.FromSeconds(15));
             
             var config = await File.ReadAllTextAsync("app.conf");
-            var conf = ConfigurationFactory.ParseString(config)
-                .WithFallback(GetSqlHocon(sqlConnectionString, sqlProviderName))
-                .WithFallback(OpsConfig.GetOpsConfig())
-                .WithFallback(ClusterSharding.DefaultConfig())
-                .WithFallback(DistributedPubSub.DefaultConfig())
-                .WithFallback(SqlPersistence.DefaultConfiguration);
-
-            var actorSystem = ActorSystem.Create("AkkaPricing", conf.BootstrapFromDocker());
-            var readJournal = actorSystem.ReadJournalFor<SqlReadJournal>(SqlReadJournal.Identifier);
-            var priceViewMaster = actorSystem.ActorOf(Props.Create(() => new PriceViewMaster()), "prices");
-
-            Cluster.Cluster.Get(actorSystem).RegisterOnMemberUp(() =>
+            using var host = new HostBuilder()
+            .ConfigureServices((hostContext, services) =>
+            {
+
+                services.AddAkka("AkkaPricing", options =>
+                {
+                    // Add HOCON configuration from Docker
+                    var conf = ConfigurationFactory.ParseString(config)                 
+                    .WithFallback(GetSqlHocon(sqlConnectionString, sqlProviderName))                
+                    .WithFallback(OpsConfig.GetOpsConfig())                 

```

Comment:
If you're using Akka.NET 1.5.2, this is no longer needed, `SplitBrainResolver` `KeepMajority` is now enabled by default.